packages/documentation-site/patternfly-docs/content/get-started/about-patternfly.md

```diff
@@ -0,0 +1,343 @@
+      <CardBody> 
+        We have a Slack workspace, where you can ask us questions and share any feedback. Just like PatternFly, our Slack channels are open for all and we encourage you to join to connect with the team.
+      </CardBody>
+      <Divider />
+      <CardFooter>
+        [Join our Slack workspace](https://www.redhat.com/mailman/listinfo/patternfly)
+      </CardFooter>
+    </Card>
+    <Card isCompact>
+      <CardTitle className="pf-v5-u-font-size-l"> Stay up to date </CardTitle>
+      <CardBody>
+        Make sure you're in the loop on important updates and discussions by signing up for PatternFly emails. We can send meeting reminders, updates, and other important information right to your inbox.
+      </CardBody>
+      <Divider />
+      <CardFooter>
+        [Sign up for our mailing list]()
+      </CardFooter>
+    </Card>
+    <Card isCompact>
+      <CardTitle className="pf-v5-u-font-size-l"> Join our meetings</CardTitle>
+      <CardBody>
+        Flyers may be spread all over the globe, but we make sure to come together to share updates and collect feedback. If you can't make it, all meetings are recorded and published on <a href="https://www.youtube.com/channel/UCqLT0IEvYmb8z__9IFLSVyQ"> our YouTube channel. </a>
+      </CardBody>
+      <Divider />
+      <CardFooter>
+        [View our Google calendar](https://calendar.google.com/calendar/embed?src=patternflyteam%40gmail.com&ctz=America%2FNew_York)
+      </CardFooter>
+    </Card>
+    <Card isCompact>
+      <CardTitle className="pf-v5-u-font-size-l"> Read our blog </CardTitle>
+      <CardBody>
+        Our team is passionate, and we have a lot to say about open source and user experience—too much to fit on our website! Our Medium publication hosts articles about PatternFly projects, industry practices, professional experience, and more. 
+      </CardBody>
+      <Divider />
+      <CardFooter>
+        [Visit our Medium publication](https://medium.com/patternfly)
+      </CardFooter>
+    </Card>
+     <Card isCompact>
+      <CardTitle className="pf-v5-u-font-size-l"> Follow us on Twitter </CardTitle>
+      <CardBody>
+        If you want to keep up with us via social media, you can find us on Twitter. We tweet meeting reminders, release announcements, community messages, and links to new Medium articles that you can add to your reading list. 
+      </CardBody>
+      <Divider />
+      <CardFooter>
+        [Follow us on Twitter](https://twitter.com/patternfly)
+      </CardFooter>
+    </Card>
+     <Card isCompact>
+      <CardTitle className="pf-v5-u-font-size-l"> Request a new feature </CardTitle>
+      <CardBody>
+        We welcome all ideas for adding or improving features! We review all requests, taking into account scope and technical constraints. Accepted requests are placed on [our feature roadmap](https://github.com/orgs/patternfly/projects/16), and we'll work with you to design and develop a solution.
```

Comment:
Can we update this roadmap link to this one instead https://github.com/orgs/patternfly/projects/16/views/2